app/assets/stylesheets/cities.scss

```diff
@@ -0,0 +1,27 @@
+h1 {
```

Comment:
for general markup styling, I would not recommend placing these in domain specific css files. I would include them in the bottom of the `application.css` files or in a `styles.scss` file that is required in `app/assets/stylesheets/application.css`